packages/dashboard/src/services/useAccountsApi.js

```diff
@@ -1,15 +1,21 @@
 import useSWR from "swr";
+import { useRouter } from "next/router";
 import { StatusCodes } from "http-status-codes";
 
-const fetcher = (url) =>
-  fetch(url).then((res) => {
+const prefix = process.env.NEXT_PUBLIC_SKYNET_DASHBOARD_URL ?? "";
```

Comment:
This env var used to be defined in `packages/dashboard/.env` which is being deleted. Is it defined anywhere else?

Reply:
Based on https://github.com/SkynetLabs/skynet-webportal/pull/1377#discussion_r755498965 this is something only used for local development. Not a show-stopper then.